pkg/models/constants.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+const (
+	// DefaultNamespace is the default namespace.
+	DefaultNamespace = "default"
+)
+
+const (
+	JobTypeService = "service"
+	JobTypeBatch   = "batch"
+	JobTypeOps     = "ops" // TODO: revisit the job naming
+	JobTypeDaemon  = "daemon"
```

Comment:
A one-line comment here will help with remembering what the different types do so our editors can show us without having to look it up in notion.

Reply:
Done in 0fc4b259977f5fc4019018abdef1346ee35b35eb